stock_batch_picking/report/batch_report.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# © 2012-2014 Nicolas Bessi, Joel Grand-Guillaume, Alexandre Fayolle
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import logging
+import time
+from os.path import commonprefix
+from openerp.report import report_sxw
+from openerp import pooler, models
+
+_logger = logging.getLogger(__name__)
+
+
+class BatchAgregation(object):
+    """ Group operations from a single batch by source and dest locations
+    """
+
+    def __init__(self, batch_id, operations_by_loc):
+        self.batch_id = batch_id
+        self.operations_by_loc = operations_by_loc
+
+    def exists(self):
+        return False
+
+    @property
+    def picker_id(self):
+        return self.batch_id.picker_id
+
+    @property
+    def batch_name(self):
+        return self.batch_id.name
+
+    @property
+    def batch_notes(self):
+        return self.batch_id.notes or u''
+
+    def __hash__(self):
+        return hash(self.batch_id.id)
+
+    def __eq__(self, other):
+        return self.batch_id.id == other.batch_id.id
+
+    def iter_locations(self):
```

Comment:
Possible to get a doc block here explaining the return? I'm having trouble following exactly what this method interface is